ott/core/potentials.py

```diff
@@ -141,7 +151,7 @@ def __init__(
 
     # we pass directly the arrays and override the properties
     # since only the properties need to be callable
-    super().__init__(f, g)
+    super().__init__(f, g, cost_fn=geom.cost_fn, cor=False)
```

Comment:
I believe, a bit like the `cost_fn` parameter in `DualPotentials`, that we should avoid setting default parameters for anything that is constructed from the output of `Sinkhorn`, otherwise we might be in the situation where the user estimates the potentials with some weights, but forgets to pass them on at a later stage and this fails silently.